ci/docker/conda-python-dask.dockerfile

```diff
@@ -25,4 +25,4 @@ COPY ci/scripts/install_dask.sh /arrow/ci/scripts/
 RUN /arrow/ci/scripts/install_dask.sh ${dask}
 
 # The Spark tests currently break with pandas >= 1.0
```

Comment:
xref https://github.com/apache/arrow/pull/9115#discussion_r553045953 we should be able to bump the pandas version to >= 1.0 in case that helps with the build failure

Reply:
yeah, you could try pandas 1.0.0, because I think both 1.1 and 1.2 will give failures with dask at the moment (my open PR has fixes for both)